cocassel/milk-buddy
# Wrapped text + added margins on export pages and reset page

Test on different emulators if possible, especially smaller emulators (i.e Nexus One)

**Testing Instructions**
1. Navigate to the receiver login and login as a receiver 
2. Collect all the containers 
3. You will be automatically re-directed to the export page
_The text at the top of the export page should have a margin (should not be stuck to the edges of the phones) around it and should not be cutoff_
4. Navigate to the reset page after the export pages
_The text at the top of the reset page should have a margin (should not be stuck to the edges of the phones) around it and should not be cutoff_
